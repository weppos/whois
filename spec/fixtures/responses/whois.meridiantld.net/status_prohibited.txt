Domain Name: 2e.tc
Domain Status: Prohibited String - Object Cannot Be Registered
Notes: This name is not allowed by the policy of this registry, and can not be registered

The data in the WHOIS database of Meridian TLD Corporation is provided by Meridian TLD Corporation for information purposes, and to assist persons in obtaining information about or related to domain name registration records. Meridian TLD Corporation does not guarantee its accuracy.By submitting a WHOIS query, you agree that you will use this data only for lawful purposes and that, under no circumstances, you will use this data to
1) allow, enable, or otherwise support the transmission of mass
   unsolicited, commercial advertising or solicitations via E-mail (spam)
or
2) enable high volume, automated, electronic processes that apply to Meridian TLD  Corporation or its systems.
Meridian TLD Corporation reserves the right to modify these terms.
By submitting this query, you agree to abide by this policy.

>>> Last update of WHOIS database: 2014-06-08T17:26:00.412Z <<<
