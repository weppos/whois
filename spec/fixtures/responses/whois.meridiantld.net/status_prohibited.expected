#domain
  %s == "2e.tc"

#domain_id
  %s == nil


#status
  %s == :reserved

#available?
  %s == false


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#registrar
  %s == nil


#nameservers
  %s %CLASS{array}
  %s == []
